internal/services/securitycenter/registration.go

```diff
@@ -33,18 +33,20 @@ func (r Registration) SupportedDataSources() map[string]*pluginsdk.Resource {
-		"azurerm_advanced_threat_protection":                      resourceAdvancedThreatProtection(),
-		"azurerm_iot_security_device_group":                       resourceIotSecurityDeviceGroup(),
-		"azurerm_iot_security_solution":                           resourceIotSecuritySolution(),
-		"azurerm_security_center_assessment":                      resourceSecurityCenterAssessment(),
-		"azurerm_security_center_assessment_metadata":             resourceArmSecurityCenterAssessmentMetadata(),
-		"azurerm_security_center_assessment_policy":               resourceArmSecurityCenterAssessmentPolicy(),
-		"azurerm_security_center_contact":                         resourceSecurityCenterContact(),
-		"azurerm_security_center_setting":                         resourceSecurityCenterSetting(),
-		"azurerm_security_center_subscription_pricing":            resourceSecurityCenterSubscriptionPricing(),
-		"azurerm_security_center_workspace":                       resourceSecurityCenterWorkspace(),
-		"azurerm_security_center_automation":                      resourceSecurityCenterAutomation(),
-		"azurerm_security_center_auto_provisioning":               resourceSecurityCenterAutoProvisioning(),
-		"azurerm_security_center_server_vulnerability_assessment": resourceServerVulnerabilityAssessment(),
+		"azurerm_advanced_threat_protection":                                resourceAdvancedThreatProtection(),
+		"azurerm_iot_security_device_group":                                 resourceIotSecurityDeviceGroup(),
+		"azurerm_iot_security_solution":                                     resourceIotSecuritySolution(),
+		"azurerm_security_center_assessment":                                resourceSecurityCenterAssessment(),
+		"azurerm_security_center_assessment_metadata":                       resourceArmSecurityCenterAssessmentMetadata(),
+		"azurerm_security_center_assessment_policy":                         resourceArmSecurityCenterAssessmentPolicy(),
+		"azurerm_security_center_contact":                                   resourceSecurityCenterContact(),
+		"azurerm_security_center_setting":                                   resourceSecurityCenterSetting(),
+		"azurerm_security_center_subscription_pricing":                      resourceSecurityCenterSubscriptionPricing(),
+		"azurerm_security_center_workspace":                                 resourceSecurityCenterWorkspace(),
+		"azurerm_security_center_automation":                                resourceSecurityCenterAutomation(),
+		"azurerm_security_center_auto_provisioning":                         resourceSecurityCenterAutoProvisioning(),
+		"azurerm_security_center_server_vulnerability_assessment":           resourceServerVulnerabilityAssessment(),
+		"azurerm_security_center_server_vulnerability_assessment_hybrid_vm": resourceServerVulnerabilityAssessmentHybridVM(),
+		"azurerm_security_center_server_vulnerability_assessment_vm":        resourceServerVulnerabilityAssessmentVM(),
```

Comment:
can we spell this out a little:

```suggestion
		"azurerm_security_center_server_vulnerability_virtual_machine_assessment":        resourceServerVulnerabilityAssessmentVM(),
```